apps/testnorge-aareg/aareg-core/src/main/java/no/nav/registre/aareg/consumer/rs/AaregstubConsumer.java

```diff
@@ -75,7 +77,7 @@ public AaregstubConsumer(
             return response.getBody();
         } else {
             log.error("AaregstubConsumer.sendTilAaregstub: Kunne ikke hente response body fra Aaregstub: NullPointerException");
```

Comment:
Jeg ville kalt denne for NullPointerException men noe med at response ikke finnes.

Reply:
Mener du å throwe f.eks ResponseNullPointerException her i tillegg til log.error, som det er gjort i eksemplet under? 😄